test_runner/regress/test_tenants.py

```diff
@@ -162,6 +164,62 @@ def test_metrics_normal_work(neon_env_builder: NeonEnvBuilder):
             f"process_start_time_seconds (UTC): {datetime.fromtimestamp(metrics.query_one('process_start_time_seconds').value)}"
         )
 
+    def query_all_safekeepers(name, filter):
+        return list(
+            chain.from_iterable(
+                map(
+                    lambda sk: sk.query_all(name, filter),
+                    sk_metrics,
+                )
+            )
+        )
+
+    crossaz_read_bytes = sum(
```

Comment:
I guess what it does, but jumps with sum/list/chain_iterable also deserve some comments.